content/en/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md

```diff
@@ -92,7 +92,7 @@ before joining the new nodes and ensuring the workloads can safely move to the n
 nodes before deleting the old nodes.
 {{< /note >}}
 
-### Modify the kubelet ConfigMap
+### Modify the kubelet ConfigMap  {#modify-kubelet-configmap}
```

Comment:
This change breaks existing bookmarks to the heading, if there are any. Why should we make the change?

Could we continue to use the existing, automatic fragment (`#modify-the-kubelet-configmap`) instead?

There might be a good reason, I just don't know what that reason is.

Reply:
I don't know why the automatic fragment (#modify-the-kubelet-configmap)  is not exist.Is this a bug?Or may I change the the fragment to the #modiy-the-kubelet-configmap to match it?